src/main/java/object/team/Country.java

```diff
@@ -15,22 +15,28 @@ public enum Team {
     private final List<Integer> maSangXCoordinates;
     private final int maSangYCoordinate;
 
-    Team(String teamName, String colorCode, List<Integer> maSangXCoordinates, int maSangYCoordinate) {
+    Country(String teamName, String colorCode, List<Integer> maSangXCoordinates, int maSangYCoordinate) {
         this.teamName = teamName;
         this.colorCode = colorCode;
         this.maSangXCoordinates = maSangXCoordinates;
         this.maSangYCoordinate = maSangYCoordinate;
     }
 
-    public boolean isSameTeam(Team team) {
-        return this.equals(team);
+    public boolean isSameCountry(Country country) {
+        return this.equals(country);
```

Comment:
enum 비교는 equals 보단 == 을 사용하세요!

```suggestion
        return this == country;
```

* 참고 : https://chan9.tistory.com/174

Reply:
감사해요!  Enum은 싱글톤 객체들이 미리 생성되어 있는 특별한 클래스이기 때문에 equals가 아닌 == 비교를 해도 같은 인스턴스를 가리키므로 가능하다는 걸 알게 되었습니다!

참고 자료도 첨부해 주셔서 감사합니다. ☺️